pkg/cluster/nodes/nodes.go

```diff
@@ -134,3 +134,14 @@ func WaitForReady(node *Node, until time.Time) bool {
 		return true
 	})
 }
+
+// helper that calls `try()`` in a loop until the deadline `until`
+// has passed or `try()`returns true, returns wether try ever returned true
+func tryUntil(until time.Time, try func() bool) bool {
```

Comment:
this is the same code, but the only user is over here now (directly above) so I moved it.